terraform/environments/digital-prison-reporting/main.tf

```diff
@@ -1068,6 +1068,61 @@ module "kinesis_cdc_domain_stream" {
   cloudwatch_logging_enabled = true
 }
 
+# Dynamo DB for DomainRegistry, DPR-306/DPR-218
+module "dynamo_tab_domain_registry" {
+  source              = "./modules/dynamo_tables"
+  create_table        = true
+  autoscaling_enabled = false
+  name                = "${local.project}-domain-registry-${local.environment}"
+
+  hash_key    = "primaryId"
+  range_key   = "secondaryId"
+  table_class = "STANDARD"
+  ttl_enabled = false
+
+  attributes = [
+    {
+      name = "primaryId"
+      type = "S"
+    },
+    {
+      name = "secondaryId"
+      type = "S"
+    },
+    {
+      name = "type"
+      type = "S"
+    }
+  ]
+
+  global_secondary_indexes = [
+    {
+      name            = "primaryId-Index"
```

Comment:
Minor - the original index names on the sandbox reflected that these are composite indexes hence `primaryId-type-index` so you might want to change the names here to match if you think that makes more sense.

Reply:
ah okay - I see, let me get that looked into